### ISSUE TITLE
Any way we can avoid nested workflow names?

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/workflows#4]
<img width="496" alt="Screenshot 2022-09-04 at 09 02 39" src="https://user-images.githubusercontent.com/11330453/188301599-f51b7ca7-f22c-457d-b780-8178cb8535ce.png">

**Comment by strengejacke** [comment]
This is new isn't it?

**Comment by IndrajeetPatil** [comment]
Yes, because we are calling workflows from another repository now.

But, even then, the workflow name should only appear twice, and not thrice.

**Comment by etiennebacher** [comment]
This doesn't really matter IMO and I doubt anyone wants to spend time on this.
